firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigGetParameterHandler.java

```diff
@@ -92,14 +102,14 @@ public ConfigGetParameterHandler(
    * @param key A Firebase Remote Config parameter key.
    */
   public String getString(String key) {
-    String activatedString = getStringFromCache(activatedConfigsCache, key);
-    if (activatedString != null) {
-      return activatedString;
+    ConfigAndValue configAndValue = getStringFromCache(activatedConfigsCache, key);
```

Comment:
Given the `config` is available as `activatedConfigsCache` it's unclear to me why we need the `ConfigAndValue` tuple, eg as opposed to:

```
public String getString(String key) {
  String activatedString = getStringFromCache(activatedConfigsCache, key);
  if (activatedString != null) {
    ConfigContainer cachedContainer = getConfigsFromCache(activatedConfigsCache);
    callListeners( ... cachedContainer);
    return activatedString;
  }
  ...
```

Reply:
That works too. I figured we're already calling getConfigsFromCache() in getStringFromCache(), so it was easier to just return it, but I guess it is ugly.